### DISCUSSION TITLE
Input/output channel mapping

### DISCUSSION THREAD

**Discussion by sajjal0912** [jamulussoftware/jamulus#3136] [General]
How can i see Input channel mapping and output channel mapping?

**Comment by pljones** [comment]
I'm not sure what you mean.

Jamulus has one or two input channels.  If you set "Mono" in "Audio Channels" in "Settings", "Audio/Network Setup":
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/6258b87d-244e-4752-9f84-8f36d2308a19)
you'll get a single mono input for both input and output.  You can switch Left/Right on the main window, under the "Reverb" fader:
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/6ea19abd-6ff1-490a-8cb6-3136127bfc78)

For "Mono in / Stereo out", you'll still be sending just the one channel to the Server for mixing but receiving back a stereo pair.

For "Stereo", you're expected to be sending a stereo pair and you'll get a stereo mix back from the Server.

If your selected Audio Device has multiple ins/outs, you'll be able to choose which to use.  If there's only a single pair, you'll not be able to choose (as it's redundant to pick from a choice of one).
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/2a69d2da-c9bc-4572-89dc-2ae1027f268c)
